components/form/autocompleted/src/index.js

```diff
@@ -143,15 +148,28 @@ export default class FormAutocompleted extends Component {
           {...this.props}
           handleSelect={this._handleSelect}
           active={active}
+          ref={node => { this.suggestList = node }}
           />
         )
       : null
   }
+  _handleOutsideClick = (event) =>
+    !this.excludeFromOutsideClick.includes(event.target) &&
+    this.setState({
+      showSuggestsList: false
+    })
 
   componentDidMount () {
     if (this.state.focus) {
       this.focusInput()
     }
+    this.excludeFromOutsideClick = [
```

Comment:
This looks like a fancy solution, but since this event will be triggered anywhere in the page i'm not sure about the performance. What do you think @carlosvillu @kikoruiz ?